test/acceptance/features/volumeMount.feature

```diff
@@ -0,0 +1,166 @@
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-backend" should be changed to "True"
+        And The env var "BACKEND_HOST" is not available to the application
+        And The env var "BACKEND_PORT" is not available to the application
+        And Content of file "/var/data/binding-backend/BACKEND_HOST" in application pod is
+            """
+            example.common
+            """
+        And Content of file "/var/data/binding-backend/BACKEND_PORT" in application pod is
+            """
+            8080
+            """
+        And Content of file "/var/data/binding-backend/MYHOST" in application pod is
+            """
+            example.common
+            """
+
+    Scenario: Binding is injected as file into application pod at default location
+        Given OLM Operator "backend" is running
+        * Generic test application "generic-app-a-d-u" is running
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+                annotations:
+                    "service.binding/host": "path={.spec.host}"
+                    "service.binding/port": "path={.spec.port}"
+            spec:
+                host: example.common
+                port: 8080
+            """
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-backend
+            spec:
+                bindAsFiles: true
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
+
+                application:
+                    name: generic-app-a-d-u
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-backend" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-backend" should be changed to "True"
+        And The env var "BACKEND_HOST" is not available to the application
+        And The env var "BACKEND_PORT" is not available to the application
+        And Content of file "/bindings/binding-backend/BACKEND_HOST" in application pod is
+            """
+            example.common
+            """
+        And Content of file "/bindings/binding-backend/BACKEND_PORT" in application pod is
+            """
+            8080
+            """
+
+    Scenario: Binding is injected as file into application pod at overriden location
```

Comment:
please put more details on how we set the path - in this case we set `mountPath` on the service binding

Reply:
Done. 172198aa20026266b5b017693826e754be0f1567